console: Document redirect URI and clarify binding (#777)
redpanda-data/documentation: docs/platform/console/single-sign-on/identity-providers/generic-oidc.mdx

> you can configure the generic OIDC provider. To do so, you first have to create an OAuth application in your identity provider

Reviewer: ```suggestion
you can configure the generic OIDC provider. To do so, create an OAuth application in your identity provider
```

    userIdentifyingClaimKey: "sub"
```

:::note

> The `userIdentifyingClaimKey` can only be used for claims that have a string value. For example it is
> not possible to match on the `groups` claim in AzureAD because the claim value is a string array.

Reviewer: ```suggestion
The `userIdentifyingClaimKey` can only be used for claims that have a string value. You cannot use it for the `groups` claim in AzureAD, for example, because the claim value is a string array.
```

:::

## Define role-bindings

> When you have setup the OIDC login configuration, you can now bind users to roles. Below you can find a sample

Reviewer: ```suggestion
After you set up the OIDC login configuration, you can bind users to roles. Below is a sample
```
